<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="surveyButton">
     <property name="text">
      <string>Survey</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="transectButton">
     <property name="text">
      <string>Transect</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="strateButton">
     <property name="text">
      <string>Strate</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="plateformButton">
     <property name="text">
      <string>Plateform</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
